src/js/components/Calendar/stories/Simple.js

```diff
@@ -15,12 +23,11 @@ class SimpleCalendar extends Component {
   render() {
     const { date } = this.state;
     return (
-      <Grommet theme={grommet}>
+      <Grommet theme={customTheme}>
```

Comment:
This change will impact all the stories that use the SimpleCalendar component to have a custom theme. It won't make it that `Simple` anymore ;)

Reply:
this was for testing sorry didn't mean to push up on the simple example. I can make a new story and revert these back ?

Reply:
sounds good